### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by W5EIM** [mi0bot/OpenHPSDR-Thetis#105]
**Describe the bug**
v2.10.3.7 Beta 2
Audil / VAC1  Gain(bd)  TX does not save.  Upon starting TheTis the previously stored value is not returned.  It defalults to 0

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Audio'
2. Click on 'VAC1'
3. Scroll down to 'TX Gain is not saved/restored.'

**Expected behaviour**
Expected behaviour is that this value is maintained

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windosl 11
 - Thetis version: v2.10.3.7 Beta 2

**Additional context**

**Comment by DH1KLM** [comment]
You have to save your tx profile in setup.

**Comment by W5EIM** [comment]
I have confirmed that I have saved the Tx profile multiple times.  And each
time I shutdown and restart TheTis the setting is reset to 0.

I have also exported the profile and confirmed that my setting is in the
saved profile so it looks like it is saved but not being restored.

This worked correctly in 3.6.

Dalton W5EIM

*Dalton Williams*
DeFever 41 MV Easy Living
20 Years & 34,800 Miles on the water
Amateur Radio W5EIM
Capital Yacht Club Member
MTOA Member
AGLCA Member
